admin/notif-services/jira.rst

```diff
@@ -78,7 +78,7 @@ Step 2: Add a Jira integration as a detector alert recipient in Observability Cl
 =================================================================================================
 
 ..
-  once the detector docs are migrated - this step may be covered in those docs and can be removed from these docs. below link to :ref:`detectors` and :ref:`receiving-notifications` instead once docs are migrated
+  once the detector docs are migrated - this step may be covered in those docs and can be removed from these docs. below link to :ref:`detectors` and :ref:`receiving-notifications` instead once docs are migrated. (Where are the docs migrating to exactly???)
```

Comment:
This line is a comment for other writers. We haven't decided if we need to move this step or keep it. Meanwhile, you can ignore it.